docs2/site/docs/getting-started/transport.md

```diff
@@ -0,0 +1,89 @@
+- `Variables` - (Optional, Inputs): Values for any Variables defined by the Operation.
+- `Extensions` - (Optional, Inputs): This entry is reserved for implementors to extend the protocol however they see fit.
+
+**Note:** the *Query* property can be null in case of [persisted queries](https://www.apollographql.com/docs/apollo-server/performance/apq/) when a client sends only SHA-256 hash of the query in *Extensions* given that corresponding key-value pair has been saved on a server beforehand
+
+
+
+
+## OperationMessage
+
+`OperationMessage` represents a message typically used by the [GraphQL-WS](https://the-guild.dev/graphql/ws) or  [graphql-transport-ws](https://github.com/apollographql/subscriptions-transport-ws/blob/master/PROTOCOL.md) WebSockets-based protocols.
+Both of these protocols are used by the [Apollo Client](https://www.apollographql.com/docs/react/) library. The message contains the following properties:
+- Id - (Optional, string): The id of the message.
+- Type - (Required, string): The type of the message. Can be one of the following:
+    #### GraphQL-WS
+    - `connection_init` - Client -> Server. Client sends this message to initialize the connection.
+    - `connection_ack` - Server -> Client. Server sends this message to acknowledge the connection.
+    - `error` - Server -> Client. Server sends this message to indicate that an error occurred.
+    - `ping` - Bidirectional. Client or Server sends this message to keep the connection alive.
+    - `pong` - Bidirectional. Client or Server sends this message to keep the connection alive.
+    - `subscribe` - Client -> Server. Client sends this message to subscribe to a GraphQL subscription.
+    - `next` - Server -> Client. Server sends this message to indicate that a new value has been received for a GraphQL subscription.
+    - `complete` - Bidirectional. This message indicate that the GraphQL subscription has completed.
+
+    #### graphql-transport-ws
+    - `GQL_CONNECTION_INIT` - Client -> Server. Client sends this message to initialize the connection.
+    - `GQL_CONNECTION_ACK` - Server -> Client. Server sends this message to acknowledge the connection.
+    - `GQL_CONNECTION_ERROR` - Server -> Client. Server sends this message to indicate that an error occurred while establishing the connection.
+    - `GQL_CONNECTION_TERMINATE` - Client -> Server. Client sends this message to terminate the connection.
+    - `GQL_START` - Client -> Server. Client sends this message to start a GraphQL operation.
+    - `GQL_DATA` - Server -> Client. Server sends this message to send the GraphQL execution result.
+    - `GQL_ERROR` - Server -> Client. Server sends this message to indicate that an error occurred during the GraphQL operation.
+    - `GQL_COMPLETE` - Server -> Client. Server sends this message to indicate that the GraphQL operation is complete.
+    - `GQL_STOP` - Client -> Server. Client sends this message to stop a running GraphQL operation.
```

Comment:
These are constants and the actual string values.  I recommend using the actual string values to match the above.

```suggestion
    - `connection_init` - Client -> Server. Client sends this message to initialize the connection.
    - `connection_ack` - Server -> Client. Server sends this message to acknowledge the connection.
    - `connection_error` - Server -> Client. Server sends this message to indicate that an error occurred while establishing the connection.
    - `ka` - Server -> Client. Server sends this message periodically to keep the client connection alive.
    - `connection_terminate` - Client -> Server. Client sends this message to terminate the connection.
    - `start` - Client -> Server. Client sends this message to start a GraphQL operation.
    - `data` - Server -> Client. Server sends this message to send the GraphQL execution result.
    - `error` - Server -> Client. Server sends this message to indicate that an error occurred during the GraphQL operation.
    - `complete` - Server -> Client. Server sends this message to indicate that the GraphQL operation is complete.
    - `stop` - Client -> Server. Client sends this message to stop a running GraphQL operation.
```

Reply:
Also `ka` was missing so I added it.